frontend/src/components/RequestFormInputs.vue

```diff
@@ -39,8 +39,8 @@
             <button
               class="btn btn-ghost btn-sm text-orange m-2"
               type="button"
-              :disabled="addTokenButtonDisabled"
-              @click="addToken"
+              :disabled="!addTokenExecutable"
```

Comment:
Same problem with `executable` naming as earlier. Can be understood a s noun. 😬 🙈